lms/static/js/edxnotes/views/notes_factory.js

```diff
@@ -7,9 +7,11 @@
         'js/edxnotes/plugins/events', 'js/edxnotes/plugins/accessibility',
         'js/edxnotes/plugins/caret_navigation',
         'js/edxnotes/plugins/store_error_handler',
-        'js/edxnotes/plugins/search_override'
+        'js/edxnotes/plugins/search_override',
+
+        'js/edxnotes/plugins/ai_summarize'
```

Comment:
What do you think of adding a trailing coma here? 
```suggestion
        'js/edxnotes/plugins/ai_summarize',
```